src/Orchard.Cms.Web/Modules/Lucene/Startup.cs

```diff
@@ -24,7 +28,8 @@ public override void ConfigureServices(IServiceCollection services)
             services.AddScoped<IContentPartFieldDefinitionDisplayDriver, ContentPartFieldIndexSettingsDisplayDriver>();
             services.AddScoped<INavigationProvider, AdminMenu>();
```

Comment:
I fixed it for this module. I have done it on other modules as it's not part of this PR and we have your bug to track it.